common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/VropsPackageStore.java

```diff
@@ -1142,16 +1142,46 @@ private void setDefaultPolicy(final Package vropsPackage, final File tmpDir) {
         }
     }
 
+	/**
```

Comment:
Should we call this conditionally if it's vROPS 8.17

Reply:
This was not done, but marked as resolved, no comment was added either

Reply:
This is part of another functionality supported by vROPs 8.12 and above